scripts/assets/test_file_exemptions.textproto

```diff
@@ -265,7 +265,10 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/player/state/testin
 exempted_file_path: "app/src/main/java/org/oppia/android/app/player/state/testing/StateFragmentTestViewModel.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/player/stopplaying/RestartPlayingSessionListener.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/player/stopplaying/StopExplorationDialogFragment.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/player/stopplaying/ProgressDatabaseFullDialogFragment.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/player/stopplaying/UnsavedExplorationDialogFragment.kt"
```

Comment:
These actually should have tests to verify that their buttons perform the expected results.

Reply:
There are tests to verify the buttons work correctly in ExplorationActivityTest

Reply:
As discussed in chat, nearly all files should have tests to verify them in isolation but we're making an exception here for this PR since the dialogs are still being tested via ExplorationActivityTest.